### ISSUE TITLE
[Request] Gem Armor Chestplate should use EMC

### ISSUE THREAD

**Issue by Robotia** [sinkillerj/ProjectE#1117]
Currently, the Gem Armor Chestplate can infinitely create nova explosions. This is a problem considering that a player could spam the explosion and destroy another player's base with ease. Instead, EMC should be used, equivalent to a nova explosive, and a cooldown timer should be included.

**Comment by williewillus** [comment]
this is another ee2 vs projecte thing. EE2 required no fuel because at the stage where this armor can be made, *any* amount of fuel would be trivial for a player to produce.

**Comment by Robotia** [comment]
@williewillus That's a good point. Cooldown timer instead?

**Comment by escman1999** [comment]
isn't projectE supposed to be EE2, but with small improvements? wouldn't a cooldown feature be a pretty big change from how EE2 was?

**Comment by Robotia** [comment]
@escman1999 It would be, but it could be implemented for server side only so it wouldn't affect single-player gameplay.

**Comment by koalalord9999** [comment]
I think all of the endgame items should use EMC, even if it's not even a drop in the pool, I think you should still be required to have a klein star to power them.

**Comment by Carbon-Ghost** [comment]
Config for cooldown or not and emc use or not :P My opinion ^^

**Comment by Robotia** [comment]
I agree with @koalalord9999 and @Carbon-Ghost. In its current state it's just way too powerful and is likely to be a major grievance for lag on many servers.

**Comment by escman1999** [comment]
isn't the Gem armor very very expensive to make though?

**Comment by koalalord9999** [comment]
@escman1999 It is, but I feel it needs to be like 100x more expensive, as you can make an infinitly fast emc generator by abusing the crap out of pedestals and watch of flowing time. Currently, I'm making 1 billion emc per hour. I will never EVER need this much emc.

**Comment by williewillus** [comment]
The same can be said of any such cost, there is really no limit

**Comment by Carbon-Ghost** [comment]
In the past, Gem armor was often disabled, if not it was because it had special plugins for stuff..

**Comment by Robotia** [comment]
If you guys are interested, I can add a cooldown feature on a fork.

**Comment by williewillus** [comment]
The cooldown timer I will do, since the fire projectile keybinding has a cooldown already, it should be simple to expand to the other keys

**Comment by escman1999** [comment]
so, just to clarify, this feature will be server-side only?

**Comment by williewillus** [comment]
config for special keybind cooldown, defaults to no cooldown

**Comment by M3DK1T** [comment]
Perhaps somebody could draft up a unique set of 'Multiplayer Friendly' Gem armor? The way EE2 had it works fine single player, but the armor is just waaaay too powerful in Multiplayer. The fact the armor never needs repairing, never requires EMC.. The entire set is very immensely powerful. The argument was made that at the point the player has this armor, they've probably got enough power to afford any and all fuel. Well, at least create that kind of mundane requirement of carrying a battery or two around. Maybe make the drain high enough that the player has to have a few spare Omega stars just to feed the entire suit for a few hours. 

This would limit the use of the armor, perhaps make the drain of EMC specific to the part and the power? Boots may have a drain of around 8 emc per second, where as something like the chest piece could be 64 emc per second.

**Comment by Robotia** [comment]
EMC usage would be nice, considering the fact that Gem Armor is almost impenetrable (except for the Katar).

**Comment by koalalord9999** [comment]
To make the near invicibilty less OP, I think using ridiculously high EMC costs for simple features would make it a bit more balanced. True, you can just carry around an omega star, but what if the simple action of healing a haunch, or heart cost 100k EMC? That means if you were technically to take 5 hearts of damage, that's 500k EMC instantly sucked out of your star (We'd need another tier of star I think...) Creative flight would cost 5k emc/t while active, and hurricane boots flight by just holding space, cost around 3k emc/t (only cause its harder to control). Explosion chest piece I have no idea only cause I never knew it was even a feature lol, probably like 65k per explosion, with a 5 second cooldown. Nightvision from helm could be pretty cheap as it's not game changing, so lets go with 500 emc/t. Step assist, who even cares if that costs anything, but if it needed to, I'd say 1k per block stepped up.

We gotta remember, once you have this armor - you can probably produce any number of EMC forever. Just using it I feel needs to be... more, grindy to keep ontop of it if you want it to be effective.

Currently the way it is, this armor is in line with Draconic Armor from Draconic Evolution, except its thousands of times easier to make (for me).

On note of the Katar, that thing is just flat out broken, I think the most damage it should do with its death aura is 100, that will kill every normal monster instantly, but take 3-4 hits to kill a boss (still broken af against bosses but whatever)

I know you're gonna read this and be like NOPE! But, thats just my suggestion on how to balance it a little bit, if you want to be unkillable, you need to fork out the emc, no matter how 'easy' it actually is.

**Comment by MaPePeR** [comment]
I disagree. Especially on the "make it more grindy" because that is the worst kind of balance. 

And the costs you are suggesting are not uneven remotely sane.

**Comment by koalalord9999** [comment]
I'd pay them xD

**Comment by MaPePeR** [comment]
You are suggesting more than a diamond per second just for night vision. 

And more than 7 per second of flight... 
It would be more interesting to see - for example - how much RF simple jet packs uses and look how much coal it would cost to create that amount of RF. 

Your emc farm can barely keep up with the requirements for night vision alone.

**Comment by koalalord9999** [comment]
Hmm, now that I look into the math of, you're right, those numbers are a bit out there.. lol maybe cut them down a bit xD
Was just a point though, if the cost outweighs the benefits, maybe it wouldn't be as broken :p

**Comment by Robotia** [comment]
Yea koala that's a bit high...@MaPePeR would you be ok with me making a fork and trying to implement this idea?

**Comment by Starwort** [comment]
Honestly, the gem armour would be less broken if you actually had to fill the klein star to make it. Right now, you can use an empty klein star to make the gem armour
